### PR TITLE
Task1-PR2

### PR DESCRIPTION
**Pull request by iamhaiderkhan**
Urls & Basic Views
Models
Basic Forms
Basic Templates

### CODE REVIEW

File: authentication/views.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from django.shortcuts import render
+# Create your views here.
+
+
+def home(request):
```
Comment (ziafazal) [inline authentication/views.py]: Can we make both of these class based views?

File: templates/home.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+{% block content %}
+<ul class="nav">
+    {% if user.is_authenticated %}
+     <li class="nav-item">
+            <a class="nav-link active" href="{% url 'logout' %}">Logout</a>
+        </li>
+
+    {% else %}
+    {% if request.path != '/auth/login/' %}
```
Comment (ziafazal) [inline templates/home.html:13]: if statement should be
```
{% if not user.is_authenticated %}
```

**Review by ziafazal — CHANGES_REQUESTED** [review]
@iamhaiderkhan see the inline comments. Please add `.pyc` files to your .gitignore so those would not be committed.

File: authentication/views.py (hunk 1 above)
Comment (iamhaiderkhan) [reply authentication/views.py]: Yes, Today i'm working on views and forms. I will covert these function views into class based views.

File: templates/home.html (hunk 2 above)
Comment (iamhaiderkhan) [reply templates/home.html:13]: I've tried but this condition didn't show the desired result.

**Pull request merged**